src/components/team/TeamItemList.tsx

```
@@ -0,0 +1,16 @@
+import TeamItem from '@components/team/TeamItem';
+import { TeamListWithApproval } from '@/types/TeamType';
+
+type TeamItemListProps = {
+  teamList: TeamListWithApproval[];
+};
+
+export default function TeamItemList({ teamList }: TeamItemListProps) {
+  return (
+    <ul className="h-full overflow-y-auto">
+      {teamList.map((team) => (
+        <TeamItem team={team} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)